collectors/python.d.plugin/spigotmc/spigotmc.chart.py

```diff
@@ -91,33 +105,30 @@ def _get_data(self):
         data = {}
         try:
             raw = self.console.command('tps')
-            # The above command returns a string that looks like this:
-            # '§6TPS from last 1m, 5m, 15m: §a19.99, §a19.99, §a19.99\n'
-            # The values we care about are the three numbers after the :
-            tmp = raw.split(':')[1].split(',')
-            data['tps1'] = float(tmp[0].lstrip(u' §a*')) * PRECISION
-            data['tps5'] = float(tmp[1].lstrip(u' §a*')) * PRECISION
-            data['tps15'] = float(tmp[2].lstrip(u' §a*').rstrip()) * PRECISION
+            match = _TPS_REGEX.match(raw)
+            if match:
+                data['tps1'] = int(match.group(1)) * PRECISION
+                data['tps5'] = int(match.group(2)) * PRECISION
+                data['tps15'] = int(match.group(3)) * PRECISION
```

Comment:
should be

```
int(float(match.group(1)) * PRECISION)
```

Reply:
Again, too much time away from Python.  I've gotten used to JS having only one numeric type for everything...